### ISSUE TITLE
Can't reference a image by relative paths in markdown front matter

### ISSUE THREAD

**Issue by patricksimonian** [bcgov/devhub-app-web#187]
**Describe the bug**
I'm unable to reference an image in my markdown front matter by relative path or absolute path

**To Reproduce**
### Sample Markdown
```markdown
---
author: patricksimonian
description: description
title: title
image: /docs/images/myimage.png || ./images/myimages.png
---
markdown content below
```
**Expected behavior**
I'd expect that when this is transformed into an `<img src="...">`, **when viewing the resource as a card**, that the path to the image is properly processed with the original source location of the markdown file/repo

ie: `<img src="https://www.github.com/owner/repo/blob/docs/images/myimage.png?raw=true">`

## Extra Info

This appears to be **working** correctly if an image is loaded via a twitter card?

**Comment by patricksimonian** [comment]
as mentioned in #217 the code has been authored and tested but not implemented. Opening a new issue to implement the file transformer plugin that was created in #217 AFTER #213 has been merged!
